main/saturn-node.js

```diff
@@ -86,7 +86,7 @@ async function start (/** @type {Context} */ ctx) {
     return
   }
 
-  childLog = ''
+  childLog = []
```

Comment:
converted to an array of lines, to get easier line based operations

Reply:
Yes please, I would do the same 👏🏻